### ISSUE TITLE
Website build is failing

### ISSUE THREAD

**Issue by ilyavolodin** [eslint/eslint.github.io#258]
Website build (more specifically algolia re-indexing) has been failing for the last 5 builds. That means that our search is 3 version behind (last successful update was for version 2.9.0). Could somebody with some experience with Ruby take a look? I don't feel right bugging @pixelastic every time we have an issue.

cc: @eslint/eslint-team

**Comment by ilyavolodin** [comment]
Some additional information. The only difference that I found between successful build and failed is versions of the installed gems:
```
kramdown 1.11.0 -> 1.11.1
listen 3.0.6 -> 3.0.7
jekyll 3.0.4 -> 3.0.5
```

The error message is `	jekyll 3.0.5 | Error:  undefined method `captures' for nil:NilClass`. So it might be due to the issue with algolia and jekyll 3.0.5. I have no idea what version of jekyll github pages are running, but maybe we should lock down gem dependencies to that version?

**Comment by pixelastic** [comment]
No worries, you're not bugging me. I'd like this search to work flawlessly.

I don't think GitHub pages has any impact on this. The Algolia plugin is only run when you do `jekyll algolia push` and GitHub pages only do `jekyll build`. It's the Travis build that is triggering the command.

I'll clone your repo and see what is going on. I am currently working on releasing an official v1 of the plugin, so I'm glad you spotted issues like that.

FYI, if you're ok with the search content being updated only once every 24 hours, you could register for our [DocSearch](https://community.algolia.com/docsearch/) crawler. It's what [React](https://facebook.github.io/react/) and [ReactNative](https://facebook.github.io/react-native/) are using for exampke. All the hassle of maintenance is on us, but the downside is that it only crawls once a day.

**Comment by ilyavolodin** [comment]
@pixelastic Thanks for looking at it! I think DocSearch makes sense for us. We update website documentation only on releases and those don't happen more then once/twice every two weeks. How do we go about doing that? Just signed up on your site?

**Comment by pixelastic** [comment]
Yep, just sign up with the eslint website address and an email and we'll configure the crawler on our end and get back to you with the custom 4 lines of JS code to add.

**Comment by ilyavolodin** [comment]
@nzakas Could you do that please? It would be easier if we can remove Travis build all together for the site.

**Comment by nzakas** [comment]
My concern with DocSearch is that results won't show up for a day. For instance, if we release some new rules, those wouldn't show up in search results until the next day. Even though we don't update all the time, when we do, that's when we would want the search results updated ASAP. Is there any way to do that?

**Comment by pixelastic** [comment]
Currently no, and that is the drawback I was talking about. The crawler is configured to run every 24 hours and there is currently no way to tell it something changed on your side.

We plan to open-source the whole thing soon, so you'll be able to run it from Travis on demand. But in your use case, this won't change much from what you already have...

I cloned your repo and run the command from my side and everything was working well. Could you point me to a failed Travis build so I can see what is actually failing?

**Comment by ilyavolodin** [comment]
@pixelastic Absolutely. Here's the link to the built history: https://travis-ci.org/eslint/eslint.github.io/builds last 7 builds are all failing with the same issue. Here's the last build that failed: https://travis-ci.org/eslint/eslint.github.io/builds/130870741

**Comment by pixelastic** [comment]
I've submitted a PR with an edit to the code that was parsing page title to a search hierarchy. This should fix the issue :)

**Comment by nzakas** [comment]
Looks like we're still having trouble: https://travis-ci.org/eslint/eslint.github.io/builds/133462573

**Comment by pixelastic** [comment]
Oh but it seems my PR wasn't merged. I did sign the CLA, but not with the correct username apparently. Let me try again.
